src/SnowBuddies.Api/Controllers/UserProfileController.cs

```diff
@@ -21,15 +25,58 @@ public UserProfileController(IUserProfileService userProfileService, IMapper map
         [ProducesResponseType(404)]
         [ProducesResponseType(400)]
         [ProducesResponseType(500)]
-        public async Task<IActionResult> GetAllUsers()
+        public async Task<IActionResult> GetAllUserProfilesAsync()
         {
-            var users = await _userProfileService.GetAllAsync();
```

Comment:
Perhaps, the variable should be "userProfiles".
var userProfiles = await _userProfileService.GetAllAsync();